Compile error in `IpynbReport.java`:
```
/mnt/teamcity/work/f0afbbc879bedfad/server/modules/platform/api/src/org/labkey/api/reports/report/python/IpynbReport.java:408: error: incompatible types: ArchiveEntry cannot be converted to TarArchiveEntry
                                 tar.putArchiveEntry(entry);
                                                     ^
```
https://teamcity.labkey.org/buildConfiguration/LabKey_2311Release_Premium_VerifyJavaBuild/2911773

--- a/gradle.properties
+++ b/gradle.properties
@@ -127,7 +127,6 @@ commonsBeanutilsVersion=1.9.4
 commonsCollectionsVersion=3.2.2
 commonsCollections4Version=4.4
 commonsCodecVersion=1.16.0
-# sync with version Tika ships
 commonsCompressVersion=1.26.0
 commonsDbcpVersion=1.4
 commonsDiscoveryVersion=0.2

